core/src/modules/ops/export.ts

```diff
@@ -184,22 +210,21 @@ export namespace ExportOps {
 
     if (destination) where.destinationId = destination.id;
 
-    if (saveExports) {
-      const [count] = await Export.update(
-        {
-          state: "pending",
-          sendAt: new Date(),
-          startedAt: null,
-          errorMessage: null,
-          errorLevel: null,
-          retryCount: 0,
-        },
-        { where }
-      );
+    return retryExport(where, saveExports);
+  }
 
-      return count;
-    }
+  /**
+   * Finds finds and retries a failed or canceled export by id
+   */
+  export async function retryFailedExportById(
```

Comment:
Please add tests for this new method/action. There are some for the batch case in `__tests__/models/export.ts`.